onnxruntime/core/providers/cuda/tensor/expand.cc

```diff
@@ -40,6 +45,8 @@ Status Expand::ComputeInternal(OpKernelContext* ctx) const {
     for (auto i = 0; i < rank; i++) {
       in_span[i] = fast_divmod(static_cast<int>(input_shape[i]));
       out_span[i] = fast_divmod(static_cast<int>(output_shape[i]));
+      // output_shape[i] won't be 0 here, it's covered in (0 == output_shape.Size())
+      // a null output will be returned for that case
```

Comment:
The fix above seems OK -- maybe don't require the detailed comment here (i.e. seems self-explanatory).

Reply:
put the comments here to help better understand why we do / output_shape[i] without 0 checking.